### PR TITLE
[13.0][FIX] stock_picking_sale_order_link: New sale line added to done picking

### PR DESCRIPTION
**Pull request by carlosdauden**
if access from picking to sale with smartbutton

@Tecnativa TT26009

### CODE REVIEW

File: stock_picking_sale_order_link/models/stock_picking.py [hunk 1]
```diff
@@ -12,4 +12,7 @@ def action_view_sale_order(self):
         of given picking.
         """
         self.ensure_one()
-        return self.sale_id.get_formview_action()
+        # Remove default_picking_id to avoid defaults get
+        # https://github.com/odoo/odoo/blob/e4d22d390c8aa8edf757e36704a9e04b2b89f115/
+        # addons/stock/models/stock_move.py#L410
+        return self.with_context(default_picking_id=False).sale_id.get_formview_action()
```
Comment (pedrobaeza) [inline stock_picking_sale_order_link/models/stock_picking.py]: Better to simply remove the key as others do, as sometimes the inexistence of the key is not the same than having it with False:
```suggestion
        ctx = self.env.context.copy()
        ctx = ctx.pop(default_picking_id, False)
        return self.with_context(ctx).sale_id.get_formview_action()
```

**Review by victoralmau — CHANGES_REQUESTED** [review]
Exist travis error related to this change (https://travis-ci.org/github/OCA/stock-logistics-workflow/jobs/733630849#L1307), maybe because `sale_id` field may no't be defined (`False` value)
What dou you think about it?

**Comment by carlosdauden** [comment]
@victoralmau travis error was solved here:
https://github.com/OCA/stock-logistics-workflow/compare/9126e11534374d0f37529004c304def59812890f..1b3c0cb452532e92073dffe03ec8c8470c6b343d

**Comment by pedrobaeza** [comment]
Sorry for my typo in the suggestion... :cry:

**Comment by carlosdauden** [comment]
Travis green

**Pull request merged**